stories/demos/CommonFeatures.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import faker from 'faker';
+import { AutoSizer } from 'react-virtualized';
+import DataGrid, { SelectColumn, Column, RowsUpdateEvent, DEFINE_SORT } from '../../src';
+
+const dateFormatter = new Intl.DateTimeFormat(navigator.language);
+const currencyFormatter = new Intl.NumberFormat(navigator.language, {
+  style: 'currency',
+  currency: 'eur'
+});
+
+function TimestampFormatter({ timestamp }: { timestamp: number }) {
+  return <>{dateFormatter.format(timestamp)}</>;
+}
+
+function CurrencyFormatter({ value }: { value: number }) {
+  return <>{currencyFormatter.format(value)}</>;
+}
+
+interface Row {
+  id: number;
+  title: string;
+  assignee: string;
+  progress: number;
+  startTimestamp: number;
+  endTimestamp: number;
+  budget: number;
+}
+
+const columns: readonly Column<Row>[] = [
+  SelectColumn,
+  {
+    key: 'id',
+    name: 'ID',
+    width: 60,
+    frozen: true,
```

Comment:
Should we add a few more columns so it is easy to demonstrate frozen columns? Currently you cannot scroll horizontally

Reply:
I just realized that we can resize to make the horizontal scroll bar to be there.